src/main/reactjs/package.json

```diff
@@ -7,7 +7,8 @@
     "tslint": "tsc --pretty --noEmit",
     "format": "prettier --check \"./src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
     "format:write": "prettier --write \"./src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "localisation:csv": "node ../../../scripts/localisation_to_csv.js",
+    "localisation:to_csv": "node ../../../scripts/localisation_to_csv.js",
+    "localisation:from_csv": "node ../../../scripts/localisation_from_csv.js",
```

Comment:
Nyt tämä tuottaa aika lukukelvotonta JSONia -> vaikea diffata muutoksia. Voisi ajaa esim. `prettier`in käännöksille?

Reply:
Meillä taitaa tällä hetkellä olla formaattina `JSON Language Features`. Saisko tuon jotenkin tän skriptin formatteriksi? Ilmeisesti jos vaihdetaan prettieriin (tulee vähän erinäköistä jsonia) ja nykyinen musta selkeämpi, tuo menis ehkä yo. tavalla lisäten eteen `prettier --write` tms.?

Reply:
Missä tällainen formaatti on siis määriteltynä? En löydä.

Reply:
Ei tartte välttämättä olla `prettier`, mutta olisi hyvä olla joku prosessi jonka seurauksena versionhallintaan päätyisi ihmisluettavaa JSONia.

Reply:
Ei ole määriteltynä missään mutta jos VSCodella valitsee json-tiedostolle `Format document with --> JSON Language Features`, tulee siitä samannäköinen kun ennen lokalisaatioiden ajoa csv:n pohjalta.

Mun puolesta prettierin määrittämä formaatti myös ok, jokseenkin ei yhtä kätsy uusien konffien kirjoittamiseen.

Reply:
Kai se konsensus dailyn perusteella oli että voisi käyttää prettieriä tässä?

Reply:
Koitin saada tällaisen ratkaisuun toimimaan, mutta ei oikein onnistu. Seuraavassa ongelmana, että node ei ota komentoriviltä annettuja parametreja vastaan:

`"localisation:from_csv": "node ../../../scripts/localisation_from_csv.js -c $localisation_file_path -c $delimiter && prettier --write \"./public/i18n/*.json\""`

Tämä https://stackoverflow.com/questions/35428618/pass-an-argument-to-an-npm-run-script-comprising-multiple-commands pohjalta.